http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeControlledResourceComponent.scala

```diff
@@ -65,30 +65,14 @@ object controlledResourceQuery extends TableQuery(new RuntimeControlledResourceT
 sealed abstract class WsmResourceType
 
 object WsmResourceType {
-  case object AzureVm extends WsmResourceType {
```

Comment:
objects removed here are not WSM resource types anymore and were unused

Reply:
`AzureVm` is a WSM resource type -- wonder why it was unused?